changelog.d/gh-7858.added

```diff
@@ -0,0 +1,24 @@
+Extract mode: users can now choose to include or exclude rules to run on, similar to `paths:`. For example,
+to only run on the rules `example-1` and `example-2`, you would write
+
+```
+rules:
+   - id: test-rule
+     mode: extract
+     rules:
```

Comment:
Not a fan of repeating the `rules:` key here, I would have preferred `extract-rules:` or something else.

Reply:
Hm, I think you're probably right. I guess it's too late now to get it into the same release :-/ I can open a PR for next release though